src/main/java/org/betonquest/betonquest/modules/config/ConfigurationFileImpl.java

```diff
@@ -36,7 +38,8 @@ public final class ConfigurationFileImpl extends ConfigurationSectionDecorator i
     private ConfigurationFileImpl(final ConfigAccessor accessor, final ConfigAccessor patchAccessor) throws InvalidConfigurationException {
         super(accessor.getConfig());
         this.accessor = accessor;
-        if (patchAccessor != null && patchConfig(patchAccessor.getConfig())) {
+        if (patchAccessor != null) {
+            patchConfig(patchAccessor.getConfig());
             try {
                 accessor.save();
```

Comment:
The previous behaviour was to only save the config, if the patcher did changed. We want to keep that behaviour